pkg/ccl/backupccl/backupbase/constants.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Licensed as a CockroachDB Enterprise file under the Cockroach Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+
+package backupbase
+
+const (
```

Comment:
I'm OK with this since we are trying to make incremental progress here.  But I think some of these constants eventually belong in a more appropriate package.  For example, many of them have to do with the layout of the backup collection itself.

Reply:
Yup I agree, some of these constants are used in test files, backupdest, debug_backup as well as manifest handling. I want to introduce `backupinfo` to better understand the dependency graph before we move these into specific packages.